cardano-crypto-tests/test/Main.hs

```diff
@@ -20,9 +21,11 @@ tests =
   -- anything, so we set a minimum of 1000.
   adjustOption (\(QuickCheckTests i) -> QuickCheckTests $ max i 1000) .
     testGroup "cardano-crypto-class" $
-      [ Test.Crypto.DSIGN.tests
+      [ 
+        Test.Crypto.DSIGN.tests
```

Comment:
This cosmetic change is unnecessary; as a general rule, I would recommend not changing the coding style / formatting conventions of a module when making only small changes to it.

Reply:
Thanks. Right. My bad. it was not properly formatted.